exes/ethercat/inc/public/tfc/ec.hpp

```diff
@@ -371,6 +376,8 @@ class context_t {
   int32_t wkc_ = 0;
   std::array<std::byte, pdo_buffer_size> io_;
   std::unique_ptr<std::thread> check_thread_;
+  tfc::confman::config<config::bus> config_{ ctx_, "ethercat" };
```

Comment:
Could you use the dbus_ connection we added to save connections here?

Reply:
Yes, it's possible but the confman constructor needs to be changed. Also the config needs to be changed to a shared_ptr and moved, please see changes.